src/jit/morph.cpp

```diff
@@ -16649,73 +16638,64 @@ void Compiler::fgMorph()
 
     objectAllocator.Run();
 
-    /* Add any internal blocks/trees we may need */
-
-    fgAddInternal();
-
-#ifdef DEBUG
-    /* Inliner could add basic blocks. Check that the flowgraph data is up-to-date */
-    fgDebugCheckBBlist(false, false);
-    /* Inliner could clone some trees. */
-    fgDebugCheckNodesUniqueness();
-#endif // DEBUG
```

Comment:
I was referring to these lines being deleted. These checks are no longer performed, right?

Reply:
Ah, right. Thanks.

What I plan to do is be more explicit about which checks run at each point. Right now it's just "all or none" and many phases or phase-like things fire off their own checks as pre or post conditions. Would like to consolidate all this into phases if possible.

`fgDebugCheckBBList(false, false)` ends up not checking very much.

Reply:
I agree that being more explicit about the checks is the right direction. Are you going to do that work in this PR or is this future work? If it's the latter I would prefer keeping the current checks as is.

Reply:
Not sure yet how to roll out this sort of change. 

It may indeed make sense to leave checking as is for now and focus on other aspects, and then come back to checking later.